docker-compose.yml

```diff
@@ -20,8 +20,8 @@ services:
     image: dpage/pgadmin4:5.3
     restart: always
     environment:
-      PGADMIN_DEFAULT_EMAIL: admin@email.com
-      PGADMIN_DEFAULT_PASSWORD: secret
+      PGADMIN_DEFAULT_EMAIL: postgres@email.com
+      PGADMIN_DEFAULT_PASSWORD: championsclub123
```

Comment:
it's crucial to have these as environment variables

Reply:
shut the fuck up these are localhost 🐱‍🐉

Reply:
you can comment this on your postgres setup if you want

Reply:
just make sure to implement it for your next changes since its not up in prod.